app/domain/authentication/authn_jwt/input_validation/parse_mandatory_claims.rb

```diff
@@ -0,0 +1,97 @@
+module Authentication
+  module AuthnJwt
+    module InputValidation
+      # Parse mandatory-claims secret value and return a validated claims list
+      ParseMandatoryClaims ||= CommandClass.new(
+        dependencies: {
+          validate_claim_name: ValidateClaimName.new(
+            deny_claims_list_value: MANDATORY_CLAIMS_DENY_LIST
+          ),
+          logger: Rails.logger
+        },
+        inputs: %i[mandatory_claims_secret_value]
+      ) do
+        def call
+          @logger.debug(LogMessages::Authentication::AuthnJwt::ParsingMandatoryClaims.new(@mandatory_claims_secret_value))
```

Comment:
I think it will be better to change mandatory_claims_secret_value to mandatory_claims 
The fact that it comes from secret(variable) shouldn't affect here - in the future we could change it.. (and should)